bedevere/stage.py

```diff
@@ -222,6 +222,9 @@ class Blocker(enum.Enum):
             thanks = BORING_THANKS
         comment = ACK.format(greeting=thanks, core_devs=core_devs)
         await gh.post(issue["comments_url"], data={"body": comment})
+        reviewers_url = pr_url + '/requested_reviewers'
```

Comment:
Maybe a comment on what this is doing/for?
```suggestion
        # Re-request reviews from core developers based on the new state of the PR.
        reviewers_url = pr_url + '/requested_reviewers'
```